modules/sdk-coin-eth/src/lib/transactionBuilder.ts

```diff
@@ -91,6 +95,7 @@ export class TransactionBuilder extends BaseTransactionBuilder {
     this._value = '0';
     this._walletOwnerAddresses = [];
     this._forwarderVersion = 0;
+    this._walletVersion = 0;
```

Comment:
Should we default to `1` for the majority of wallet creations?

Reply:
Actually majority of ethlike coins are using wallet version 0, just polygon (the new addition) is going to use version 1 alongwith eth itself, hence have set the default version to 0

Reply:
Infact all the ethlike coins that are using account lib don't even have a version 1 at this point

Reply:
Awesome, thanks!